### PR TITLE
Feat(CX-3420): MyC Artwork Page - display new field Notes

### PR DESCRIPTION
**Pull request by lordkiz**
The type of this PR is: **Feat**

This PR solves [CX-3420]
This PR solves [CX-3424]

### Description
- This PR adds Notes to a MyCollection Artwork
- 

https://user-images.githubusercontent.com/18648835/219332379-4b3b42d6-6127-4679-9108-f07c1a6b9995.mov

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3420]: https://artsyproduct.atlassian.net/browse/CX-3420?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3424]: https://artsyproduct.atlassian.net/browse/CX-3424?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MounirDhahri — APPROVED** [review]
Nice! thanks for the thorough tests

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSidebar/MyCollectionArtworkSidebarMetadata.tsx [hunk 1]
```diff
@@ -66,8 +77,30 @@ export const MyCollectionArtworkSidebarMetadataFragmentContainer = createFragmen
...
+  truncateLimit = 0,
+}: {
+  label: string
+  value?: string | null
+  truncateLimit?: number
+}) => {
   const emptyValue = "----"
+  const [expanded, setExpanded] = useState(false)
+  const [modalOpen, setModalOpen] = useState(false)
+  const isMobile = __internal__useMatchMedia(themeProps.mediaQueries.xs)
```
Comment (olerichter00) [inline src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSidebar/MyCollectionArtworkSidebarMetadata.tsx:92]: **issue:** `__internal__useMatchMedia` is marked as internal and we should avoid using it ([Note why it should be avoided](https://github.com/artsy/force/blob/main/src/Utils/Hooks/useMatchMedia.ts#L4)). I think it should be possible to use `Media` instead.

Comment (olerichter00) [reply src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSidebar/MyCollectionArtworkSidebarMetadata.tsx:92]: @damassi what do you think?

**Review by olerichter00 — APPROVED** [review]
Looks good 👍 

I've only added a comment about using `__internal__useMatchMedia`.

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSidebar/MyCollectionArtworkSidebarMetadata.tsx (hunk 1 above)
Comment (lordkiz) [reply src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSidebar/MyCollectionArtworkSidebarMetadata.tsx:92]: Thanks @olerichter00  sorry I missed this comment. In this case I am using isMobile to conditionally toggle in the `toggle` function. It is not directly used for UI rendering or to manipulate responsiveness. I suppose that should be fine.

**Pull request merged**